src/engraving/dom/undo.cpp

```diff
@@ -3557,31 +3557,6 @@ void ChangeTieJumpPointActive::flip(EditData*)
     m_active = oldActive;
 }
 
-FretLinkHarmony::FretLinkHarmony(FretDiagram* diagram, Harmony* harmony, bool unlink)
-{
-    m_fretDiagram = diagram;
-    m_harmony = harmony;
-    m_unlink = unlink;
-}
-
-void FretLinkHarmony::undo(EditData*)
-{
-    if (m_unlink) {
-        m_fretDiagram->linkHarmony(m_harmony);
```

Comment:
This function sets the harmony's z value and applies some alignment changes. I think we still need to do these things when adding and removing harmony from fret diagrams.